sys/zuse.hoon

```diff
@@ -243,36 +243,35 @@
     ==
   ::
   ::
-  ++  constitution
+  ++  azimuth
     |%
-    ++  hull
+    ++  point
```

Comment:
Bunted here: https://github.com/urbit/urbit/blob/release-candidate/vere/dawn.c#L316. It doesn't look like the structure changes are themselves significant.